test/e2e/suites/conformance/conformance_test.go

```diff
@@ -66,15 +64,6 @@ var _ = Describe("conformance tests", func() {
 			Expect(err).NotTo(HaveOccurred())
 		}
 
-		// Starting with Kubernetes v1.25, the kubetest config file needs to be compatible with Ginkgo V2.
-		v125 := semver.MustParse("1.25.0-alpha.0.0")
-		v, err := semver.ParseTolerant(kubernetesVersion)
-		Expect(err).NotTo(HaveOccurred())
-		if v.GTE(v125) {
-			// Use the Ginkgo V2 config file.
-			e2eCtx.Settings.KubetestConfigFilePath = getGinkgoV2ConfigFilePath(e2eCtx.Settings.KubetestConfigFilePath)
-		}
-
```

Comment:
Not worth re-doing now, but when we're reverting a commit can we do it in a separate commit? i.e. The outcome of `git revert`? It just makes it obvious exactly why a change was made both during review and when spelunking in git in the future without having to refer to the PR.